deploy.sh

```diff
@@ -1,7 +1,12 @@
 #!/bin/bash
 set -ex
 
+if [ "$SKIP_DEPLOY_KUBECTL" == "true" ] || [ "$KUBECTL_SKIP_DEPLOY" == "true" ]; then
+  echo "KUBECTL_SKIP_DEPLOY is set.  Skipping."
+  exit 0
```

Comment:
I see the `kubectl` doesn't have any schema like in the other plugins. Why is that?

How would this setting work then for Ansible plugin:  
```yaml
  options:
    skip-deploy: false
```

what's the flow?